content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,200 @@ aliases:
+* Configure which percentile aggregation you want to send to Datadog with the `histogram_percentiles` parameter in your [datadog.yaml configuration file][3]. By default, only the `95pc` percentile is sent out to Datadog.
+
+[1]: /developers/metrics/types/?tab=histogram
+[2]: /developers/metrics/types/?tab=histogram#metric-submission-types
+[3]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+{{% /tab %}}
+{{% tab "Distribution" %}}
+
+**A DISTRIBUTION metric generates by default five custom metrics** for each unique combination of metric name, host, and tag values to represent the global statistical distribution of values. These 5 custom metrics represent server-side aggregations of `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
+
+The number of unique tag value combinations submitted for a DISTRIBUTION metric with this tagging scheme is **4**.
+
+* `host:A`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:400`
+* `host:B`, `endpoint:Y`, `status:200`
+
+The number of custom metrics from a [DISTRIBUTION metric][1] is five times the unique combination of metric name, host, and tag values. This results in `request.Latency` reporting a total of **5*4 = 20 custom metrics**:
+
+##### Customized Tagging
+
+You can control over [which tag combination][2] aggregations are performed for any DISTRIBUTION metric. Let's say that you want to keep only the `endpoint` and `status` tags associated with your distribution, this leaves the following 3 unique tag combinations:
+
+* `endpoint:X`, `status:200`
+* `endpoint:X`, `status:400`
+* `endpoint:Y`, `status:200`
+
+The number of custom metrics from a [DISTRIBUTION metric][1] is five times the unique combination of metric name, host, and tag values. This results in `request.Latency` reporting a total of **5*3 = 15 custom metrics** with the tagging customisation .
+
+##### Percentile Aggregations
+
+**Datadog stores 5 custom metrics for each potentially queryable tag value combination** to provide you with globally accurate percentiles: `p50`, `p75`, `p90`, `p95`, and `p99`. Suppose you have [added percentile aggregations][3] for `request.Latency` for the set of tags `endpoint` and `status` with the same tag dependency seen earlier. The number of queryable tag value combination is **8**:
+
+* `endpoint:X`, `status:200`
+* `endpoint:X`, `status:400`
+* `endpoint:Y`, `status:200`
+* `endpoint:X`
+* `endpoint:Y`
+* `status:200`
+* `status:400`
+* `*`
+
+This results in `request.Latency` with percentile aggregation enabled reporting an extra of **5*8 = 40 custom metrics**.
+
+**Note**: Is counted as queryable, tag value combinations that actually appear in your data. Since the combination { `endpoint:Y`, `status:400` } was never submitted in your data, this combination won’t be queryable and won’t count towards your custom metric count.
```

Comment:
```suggestion
**Note**: Is counted as queryable, tag value combinations that actually appear in your data. Since the combination { `endpoint:Y`, `status:400` } was never submitted in your data, this combination won’t be queryable and won’t count towards your custom metric count.
```

the first sentence is missing a subject. what is counted as queryable?

Reply:
"tag value combination", updating the sentence accordingly